source/_integrations/zha.markdown

```diff
@@ -777,7 +829,14 @@ When reporting potential bugs related to the ZHA integration on the issues track
 3. If the issue is related to a specific Zigbee device, provide both the **Zigbee Device Signature** and the **Diagnostics** information.
      - Both the **Zigbee Device Signature** and the **Diagnostics** information can be found under {% my integrations title="**Settings** > **Devices & services**" %}. Select the **Zigbee Home Automation** integration. Then, select **Configure** > **Devices** (pick your device). Select **Zigbee Device Signature** and **Download Diagnostics**, respectively.
 
-Note: Please also make sure you give it your best effort to follow the recommended best practices for avoiding both [pairing/connection difficulties](#best-practices-to-avoid-pairingconnection-difficulties) and [Zigbee interference](#zigbee-interference-avoidance-and-network-rangecoverage-optimization), (which helps free up time for developers).
+{% tip %}
+When troubleshooting, please review the following sections on this page before reaching out for support:
```

Comment:
```suggestion
For troubleshooting, read the following sections on this page. They provide information on improving your Zigbee network performance.
```
In instructions, we usually [don't use please](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please)